### PR TITLE
build: Add cloudbuilder configuration

### PR DESCRIPTION
**Pull request by EvanPurkhiser**
I copied this straight out of craft

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        # Need to pull the image first due to Kaniko
+        docker pull us.gcr.io/$PROJECT_ID/chartcuterie:$COMMIT_SHA
+        echo "$$DOCKER_PASSWORD" | docker login --username=sentrybuilder --password-stdin
+        docker tag us.gcr.io/$PROJECT_ID/chartcuterie:$COMMIT_SHA getsentry/chartcuterie:$SHORT_SHA
+        docker push getsentry/chartcuterie:$SHORT_SHA
+        docker tag us.gcr.io/$PROJECT_ID/chartcuterie:$COMMIT_SHA getsentry/chartcuterie:$COMMIT_SHA
+        docker push getsentry/chartcuterie:$COMMIT_SHA
+        docker tag us.gcr.io/$PROJECT_ID/chartcuterie:$COMMIT_SHA getsentry/chartcuterie:latest
+        docker push getsentry/chartcuterie:latest
+timeout: 960s
```
Comment (jasonious) [inline cloudbuild.yaml:34]: this is a lot of seconds

**Review by jasonious — APPROVED** [review]

File: cloudbuild.yaml (hunk 1 above)
Comment (EvanPurkhiser) [reply cloudbuild.yaml:34]: Copied from craft

**Comment by jasonious** [comment]
approved based on "this is net new and won't break anything production"

**Comment by EvanPurkhiser** [comment]
🙏 ty @jasonious

**Pull request merged**